Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -78,75 +108,514 @@ public static void ShowWindow()
             }
         }
 
-        public static MixedRealityProjectConfiguratorWindow Instance { get; private set; }
+        private void OnGUI()
+        {
+            MixedRealityInspectorUtility.RenderMixedRealityToolkitLogo();
+            if (currentStage != ConfigurationStage.Done)
+            {
+                EditorGUILayout.LabelField("Welcome to MRTK!", MixedRealityStylesUtility.BoldLargeTitleStyle);
```

Comment:
```suggestion
                GUILayout.Label("Welcome to MRTK!", MixedRealityStylesUtility.BoldLargeTitleStyle);
```
EditorGUILayout causes the text to be cut off oddly, use the following to fix the display issues